utilities/helper/accelerate.py

```diff
@@ -685,11 +691,16 @@ def main():
                 # process, other than tell the calling program
                 # whether other signals can be sent
                 os.kill(daemon_pid, 0)
-            except OSError, e:
+            except OSError:
+                e        = get_exception()
+                message  = 'the accelerate daemon appears to be running'
```

Comment:
not really a reason to avoid merge, but it would be better to compose the message inside the if (only when we use it)